docs/manual/layout/site-structure/configure-pages.en.md

```diff
@@ -149,6 +149,34 @@ redirect to the website root of the browser's language (or the fallback language
 without any other parameters. You can exclude certin (or all) website roots from this automatic redirect through this setting.
 
 
+### Website settings
+
+{{< version "4.9" >}}
+
+**Favicon:** This allows you to select a favicon for the `/favicon.ico` URL of your domain. This is especially useful for multidomain setups
+so that you can easily serve different favicons for different domains from within the same Contao instance, since you can only have one
+_physical_ `favicon.ico` file in your document root otherwise. This will enable you to show the correct favicon per domain, if any non-HTML 
+resources are displayed in the browser directly (like images or PDFs, etc.).
+
+{{% notice "warning" %}}
+Keep in mind that this will not work if you already have a physical `favicon.ico` file in your document root, as the web server will then
+serve said file directly. Make sure to delete that file before trying to use this feature.
+{{% /notice %}}
+
+{{% notice "info" %}}
+Keep in mind that this feature will not add any additional meta tags to the HTML output.
+{{% /notice %}}
+
+**Custom robots.txt content:** This allows you to define the content of the `/robots.txt` URL of your domain. This is especially useful for 
+multidomain setups, since you can only have one _physical_ `robots.txt` file in your document root otherwise. This will enable you to define 
```

Comment:
```suggestion
multi-domain setups, since you can only have one _physical_ `robots.txt` file in your document root otherwise. This will enable you to define 
```